tasks/common.yml

```diff
@@ -7,7 +7,6 @@
     file:
       dest: "{{ sensu_config_path }}/conf.d"
       state: directory
-      recurse: true
```

Comment:
Why does this need to be removed? If you're relying on the package to create sensu_config_path first, I wouldn't trust that to be the case.